src/mame/drivers/missb2.cpp

```diff
@@ -16,15 +16,13 @@ written, so it may be normal behaviour.
 
 #include "emu.h"
 #include "includes/bublbobl.h"
-
```

Comment:
Why are you removing the blank line between PCH/module header and device headers?